GOVERNANCE.md

```diff
@@ -95,15 +95,6 @@ the maintainers per the voting process below.
 * In general continue to be willing to spend at least 25% of ones time working on Rook (~1.25
```

Comment:
To be clear, I know you can't measure everyone's time, but this doesn't seem like a good measurement in itself to know if people are contributing. Other ideas of measurements can come from activity in PR, code contributions, etc.

Reply:
I'm not really sure if there is a way to programmatically measure time spent on Rook. I don't know how we'd measure things like time spent writing code, time spent reading PRs, or time spent answering questions on Slack. It seems to me that this would have to be self-managed/self-reported.

One of the related concerns here is deciding what "time spent on Rook" really means. I think there is a hidden requirement here that the time spent is deemed in some way useful to the Rook project. If I were to spend the next several months testing Rook on various clusters and hardware and doing benchmarking but I don't contribute that information back into the project, I'm obviously not spending time "on" Rook even thought I'm spending time "with" Rook.

In the current reality, I think this isn't a self-determined thing at all but something determined by the discretion of others. "Does Blaine appear to spend 1/4 of their time working on Rook (or a Rook backend) in a useful manner?"

Reply:
There is no tool and no formal tracking being calculated or reported.